assets/component-slider.css

```diff
@@ -226,6 +262,23 @@ slider-component.slider-component-full-width {
   .slider.slider--desktop.grid--collapsed .slider__slide:first-child {
     --collapsed-shadow-offset-x: 0px;
   }
+
+  .slider.slider--desktop.grid.grid--collapsed-x .slider__slide.grid__item {
+    --top-left-radius: 0px;
+    --top-right-radius: 0px;
+    --bottom-right-radius: 0px;
+    --bottom-left-radius: 0px;
+  }
+
+  .slider.slider--desktop.grid.grid--collapsed-x .slider__slide.grid__item:first-child {
+    --top-left-radius: var(--border-radius);
+    --bottom-left-radius: var(--border-radius);
+  }
+
+  .slider.slider--desktop.grid.grid.grid--collapsed-x .slider__slide.grid__item:last-child {
+    --top-right-radius: var(--border-radius);
+    --bottom-right-radius: var(--border-radius);
+  }
 }
```

Comment:
Noticed some issues with slider items when there is no vertical spacing but there is some horizontal one: 
[collection list as cards or standard](https://screenshot.click/22-37-q4map-go8gi.png)

[blog post slider video](https://screenshot.click/22-35-ovk3w-ffu61.mp4)

[multicolumn video](https://screenshot.click/22-26-d0vbe-1dyhg.mp4)

Reply:
Oh good catch. Added another slider override to each breakpoint. This case got missed because sliders are only concerned with x spacing so I forgot to make sure the original radius was applied when collapsed base styles were still applied by a y-only configuration.